src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+/* stylelint-disable color-hex-case */
+/* stylelint-disable string-quotes */
+html {
+  font-family: 'Roboto', Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.body__navigation {
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+}
+
+.logo__img {
+  display: block;
+  margin: 0 10px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  height: 60px;
+  line-height: 60px;
+  margin: 0 10px;
```

Comment:
use margin for only one side: `margin-right` or `margin-right`, and then set it to zero for first or last child according to the choice